README.md

```diff
@@ -6,10 +6,8 @@
 
 To build the site for previewing, you'll need to set the following environment variables:
 
-```
-DISCOVERDOTNET_GITHUB_TOKEN
-DISCOVERDOTNET_MEETUP_TOKEN
-```
+* `DISCOVERDOTNET_GITHUB_TOKEN`, which can be obtained via creating a [personal access token](https://github.com/settings/tokens) with `public_repo` permissions.
+* `DISCOVERDOTNET_MEETUP_TOKEN`, which can be obtained by visiting [the Meetup API key page](https://secure.meetup.com/meetup_api/key/).
```

Comment:
Very nice, thanks for the clarification - hopefully you won't be the last to work on the site and this will help those that follow :smile: